### ISSUE TITLE
CIS Benchmark -- Error building controller: unknown provider when using containerd.io as CRI

### ISSUE THREAD

**Issue by busyboy77** [rancher/dashboard#5899]
**Setup**

- Rancher version:

v2.6.4

- Browser type & version:

Version 101.0.4951.54 (Official Build) (64-bit)

**Describe the bug**

CIS Benchamrk app suite is always in a CrashLoopBackOff State when deployed on a K8s Cluster with Containerd as CRI.
the error from the CIS-operator pod is something like

```
# kubectl logs -f pod/cis-operator-54865c6788-wgrtc  -n  cis-operator-system
time="2022-05-10T10:39:11Z" level=info msg="Starting CIS-Operator"
time="2022-05-10T10:39:11Z" level=fatal msg="Error building controller: unknown provider"
```

**To Reproduce**
Deploy the K8s Cluster using Kubeadm and Containerd as  CRI 
deploy the Rancher
Deploy the CIS-Benchmark from the Apps & Marketplace.

The CIS-Operator pod is always in CrashLoopBackOff State with above given error message.

**Result**

**Expected Result**

**Screenshots**

**Additional context**

**Detailed Description**

**Context**

**Comment by technerd4357** [comment]
Yes, same issue with containerd (RuntimeVersion:  1.4.9) and k8s v1.21.7.

**Comment by nwmac** [comment]
This does not appear to be a UI issue.

I'm going to close this since its quite old. Please feel free to re-open and I will transfer it to the `rancher/rancher` repository for further investigation.
